sound/soc/sof/ipc4-topology.h

```diff
@@ -55,6 +55,9 @@
 
 #define SOF_IPC4_INVALID_NODE_ID	0xffffffff
 
+/* FW requires minimum 2ms DMA buffer size */
+#define SOF_IPC4_MIN_DMA_BUFFER_SIZE	2
```

Comment:
is this really 2ms or 2x the system tick?

I see no reason why 2ms would be required, there's nothing in hardware that requires this.

can be updated with a fixup